Formula/spades.rb

```diff
@@ -3,16 +3,15 @@ class Spades < Formula
 
   desc "De novo genome sequence assembly"
   homepage "https://cab.spbu.ru/software/spades/"
-  url "https://cab.spbu.ru/files/release3.15.5/SPAdes-3.15.5.tar.gz"
-  mirror "https://github.com/ablab/spades/releases/download/v3.15.5/SPAdes-3.15.5.tar.gz"
+  url "https://github.com/ablab/spades/releases/download/v3.15.5/SPAdes-3.15.5.tar.gz"
   sha256 "155c3640d571f2e7b19a05031d1fd0d19bd82df785d38870fb93bd241b12bbfa"
   license "GPL-2.0-only"
+  head "https://github.com/ablab/spades.git", branch: "spades_3.15.5"
```

Comment:
```suggestion
  head "https://github.com/ablab/spades.git", branch: "spades_#{version}"
```

As mentioned in the previous comment, what do we think about simply interpolating `version` here? Due to how upstream does branches, I'm not convinced that the branch name will be kept up to date with each release if it requires manual intervention (i.e., it could be missed when `brew bump-formula-pr` is used).